debug.h

```diff
@@ -3,6 +3,15 @@
 
 #include "lib/types.h"
 
+#ifndef FIO_PRINTF_FORMAT
+#if defined(__MINGW32__)
+#include <stdio.h>
```

Comment:
Please use something like the code below and leave #include <stdio.h> out:
~~~
#if defined(__clang__)
#define FIO_PRINTF_FORMAT printf
#elif __USE_MINGW_ANSI_STDIO
#define FIO_PRINTF_FORMAT gnu_printf
#elif defined(__MINGW32__)
#define FIO_PRINTF_FORMAT ms_printf
#else
#define FIO_PRINTF_FORMAT __printf__
#endif
~~~

Reply:
Ah that is friendlier than include of stdio.h...

Reply:
OK I've done this but on another branch.